sky/shell/tracing_controller.cc

```diff
@@ -33,7 +108,22 @@ void TracingController::StartTracing() {
   StartBaseTracing();
 }
 
-void TracingController::StopTracing(const base::FilePath& path) {
+void TracingController::StopTracing(const base::FilePath& path,
+                                    bool terminateLoopWhenDone) {
+  auto runner = base::MessageLoop::current()->task_runner();
```

Comment:
There's no reason to get the task_runner here.  You can PostTask directly on `base::MessageLoop::current()`